tests/testthat/_jags/data.R

```diff
@@ -0,0 +1,14 @@
+"betaZ_mean" <- c(0, 1)
```

Comment:
is this file needed? wondering because in the test script you pass this data via list

Reply:
Yes, actually, none of these 2 `.R` is needed. I added it because I though someone maye use these for direct JAGS access if needed...

Reply:
then please remove